### PR TITLE
Added Bellows.

### PR DESCRIPTION
**Pull request by LightningSh0ck**
And fixed a mistake I previously made while IJ made other passive-aggressive code format changes...

### CODE REVIEW

File: src/main/java/net/dries007/tfc/objects/blocks/BlocksTFC.java [hunk 1]
```diff
@@ -396,7 +396,8 @@ public static void registerBlocks(RegistryEvent.Register<Block> event)
         // todo: sluice
         // todo: quern
         // todo: loom
-        // todo: bellows
+        inventoryItemBlocks.put(register(r, "bellows", new BlockBellows(), CT_MISC), ItemBlockTFC.class);
+        ;
```
Comment (LightningSh0ck) [inline src/main/java/net/dries007/tfc/objects/blocks/BlocksTFC.java]: I really don't get why IJ did place that semicolon...

Comment (alcatrazEscapee) [reply src/main/java/net/dries007/tfc/objects/blocks/BlocksTFC.java]: Don't review your own PR... there is no point. Just fix what needs fixing.

Comment (LightningSh0ck) [reply src/main/java/net/dries007/tfc/objects/blocks/BlocksTFC.java]: Do you really want me to do an entire commit just to erase that semicolon?

**Comment by alcatrazEscapee** [comment]
This has been wrapped up into the torches / fire pit PR as per request.

**Pull request closed without merging**